setup.py

```diff
@@ -56,9 +56,13 @@
             'pyota-cli=iota.bin.repl:main',
         ],
     },
+    
+    # The filter module is no longer maintained and does not support Python 3.7
+    # phx-filters is a fork that supports 3.7 and 3.8 but not 2.7
 
     install_requires=[
-        'filters',
+        'filters; python_version < "3.0"',
+        'phx-filters; python_version >= "3.5"',
```

Comment:
👍🏻 Good thinking!!  Should we make the first one `filters; python_version < "3.5"`?

Reply:
Yes. Although we do not officially support these versions, it may be better to have no gaps